src/components/Timeline.tsx

```diff
@@ -98,10 +98,9 @@ interface TimelineProps {
 }
 
 const Timeline: VoidComponent<TimelineProps> = (props) => {
-  const route = () => props.route
   // TODO: align to first camera frame event
   const [markerOffsetPct, setMarkerOffsetPct] = createSignal(0)
-  const [duration] = createResource(route, (route) => getRouteDuration(route)?.asSeconds() ?? 0, { initialValue: 0 })
+  const duration = () => getRouteDuration(props.route)?.asSeconds() ?? 0
```

Comment:
I think anything that's not asynchronous doesn't benefit from `createResource`, right? Let's convert them away to funcs/memos if so

Reply:
The only benefit you get is that route was known not to be undefined in the right-hand side of createResource, and maybe you can use `.latest` and stuff? But yeah, there's lots more of these